opentelemetry-appender-tracing/benches/log-attributes.rs

```diff
@@ -0,0 +1,271 @@
+                            field2 = "field2",
+                            field3 = "field3",
+                            field4 = "field4",
+                            field5 = "field5",
+                            field6 = "field6",
+                            field7 = "field7",
+                            field8 = "field8",
+                            field9 = "field9",
+                            field10 = "field10",
+                            message = "Unable to process checkout."
+                        );
+                    }
+                    11 => {
+                        error!(
+                            name : "CheckoutFailed",
+                            field1 = "field1",
+                            field2 = "field2",
+                            field3 = "field3",
+                            field4 = "field4",
+                            field5 = "field5",
+                            field6 = "field6",
+                            field7 = "field7",
+                            field8 = "field8",
+                            field9 = "field9",
+                            field10 = "field10",
+                            field11 = "field11",
+                            message = "Unable to process checkout."
+                        );
+                    }
+                    12 => {
+                        error!(
+                            name : "CheckoutFailed",
+                            field1 = "field1",
+                            field2 = "field2",
+                            field3 = "field3",
+                            field4 = "field4",
+                            field5 = "field5",
+                            field6 = "field6",
+                            field7 = "field7",
+                            field8 = "field8",
+                            field9 = "field9",
+                            field10 = "field10",
+                            field11 = "field11",
+                            field12 = "field12",
+                            message = "Unable to process checkout."
+                        );
+                    }
+                    _ => {
+                        // Fall back to 10 attributes for any higher number
+                        error!(
+                            name : "CheckoutFailed",
+                            field1 = "field1",
+                            field2 = "field2",
+                            field3 = "field3",
+                            field4 = "field4",
+                            field5 = "field5",
+                            field6 = "field6",
+                            field7 = "field7",
+                            field8 = "field8",
+                            field9 = "field9",
+                            field10 = "field10",
+                            message = "Unable to process checkout."
+                        );
+                    }
+                }
+            });
+        });
+    });
```

Comment:
nit - courtesy AI:
```rust
let mut fields = vec![("name", "CheckoutFailed"), ("message", "Unable to process checkout.")];

for i in 1..=num_attributes {
    fields.push((format!("field{}", i).as_str(), format!("field{}", i).as_str()));
}

error!(fields.iter().map(|(k, v)| (*k, *v)).collect::<Vec<_>>());
```

Reply:
I don;t think that is doing what we want it to do. 

We could potentially create a macro to achieve it, but the current one (though involves copy-paste) is quite easy to read and reason about.